reconcile/openshift_saas_deploy_trigger_base.py

```
@@ -117,7 +125,7 @@ def setup(
     saas_files = queries.get_saas_files()
     if not saas_files:
         logging.error("no saas files found")
-        return None, None, True
+        sys.exit(ExitCodes.ERROR)
```

Should we raise an exception instead?